math/libfaiss/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libfaiss
+version             20180214
+categories          math
+license             BSD
+platforms           darwin
+maintainers         fb.com:hoss \
+                    fb.com:matthijs
```

Comment:
The maintainers line should probably be more of the form

`maintainers {@foo1 bar.com:foo1} {@foo2 bar.com:foo2} openmaintainer`

where the `@foo1` bit is the github ID of the maintainer in question associated with the email. Unless you have good reason to insist on doing it otherwise, `openmaintainer` is best because it allows others in the project to submit trivial fixes without your explicit review.